spec/e2e/features/listings/listing.feature

```diff
@@ -2,10 +2,16 @@ Feature: Listings
     As a web user
     I should be able to view a listing
 
-    Scenario: Attempting to go to a listing page using an invalid ID
-      Given I try to go to a listing page with an invalid ID
-      Then I should be redirected to the welcome page
+    # Scenario: Attempting to go to a listing page using an invalid ID
+    #   Given I try to go to a listing page with an invalid ID
+    #   Then I should be redirected to the welcome page
 
-    Scenario: Attempting to go to the Ownership listings page
-      Given I try to go to the Ownership listings page
-      Then I should see available units
+    # Scenario: Attempting to go to the Ownership listings page
+    #   Given I try to go to the Ownership listings page
+    #   Then I should see available units
+
+    Scenario: Going to ownership listings from the welcome page
```

Comment:
for now, this will only work if SHOW_OWNERSHIP_LISTINGS is set to true, which might not be true on our production tests 🤔.

Reply:
I talked to catherine and this should actually be OK